samples/bluetooth/mesh/light/README.rst

```diff
@@ -132,16 +132,16 @@ This sample is split into the following source files:
 DFU configuration
 =================
 
-To enable the DFU feature for the supported development kits, set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
+To enable the DFU feature for the supported development kits, set :makevar:`FILE_SUFFIX` to ``dfu`` when building the sample.
 For example, when building from the command line, use the following command:
 
   .. code-block:: console
 
-     west build -b <BOARD> -p -- -DEXTRA_CONF_FILE="overlay-dfu.conf"
+     west build -b <BOARD> -p -- -DFILE_SUFFIX=dfu
 
-The configuration overlay :file:`overlay-dfu.conf` enables the DFU feature.
-To review the required configuration alterations, open and inspect the :file:`overlay-dfu.conf` file.
-For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+The configuration files with the suffix ``dfu`` enable the DFU feature.
+To review the required configuration alterations, open and inspect the files :file:`prj_dfu.conf` and :file:`sysbuild_dfu.conf`.
+For more information about using file suffixes, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
```

Comment:
Would this page fit better to reference to? https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/develop/application/index.html#application-file-suffixes
Either replacing the existing or in addition.
Or like this:
```suggestion
For more information about using the :ref:`file suffixes <zephyr:application-file-suffixes>`, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
```